aws-predictions/src/main/java/com/amplifyframework/predictions/aws/configuration/IdentifyLabelsConfiguration.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws.configuration;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplifyframework.predictions.aws.NetworkPolicy;
+import com.amplifyframework.predictions.models.LabelType;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Configures the behavior for identifying labels.
+ */
+public final class IdentifyLabelsConfiguration {
+    private static final String CONFIG_NAME = "identifyLabels";
```

Comment:
/md This already lives immediately under an `"identify"`, right? Can it just be:
```
    "identify": {
        "labels": {
        }
    }
```
instead of :
```
    "identfiy": {
        "identifyLabels": {
        }
    }
```

Reply:
discussed; existing CLI behavior